ci-operator/step-registry/storage/conf/csi-gcp-pd/storage-conf-csi-gcp-pd-commands.sh

```
@@ -4,7 +4,15 @@ set -o nounset
 set -o pipefail
 
 cd /go/src/github.com/openshift/gcp-pd-csi-driver-operator
-if [ "${COMPUTE_DISK_TYPE}" == "hyperdisk-balanced" ]; then
+if [ "${IMAGE_VOLUME_SNAPSHOT_MANIFEST:-}" = "image-snapshot" ]; then
+    echo "Copying image-snapshot-manifest.yaml to ${SHARED_DIR}/${TEST_CSI_DRIVER_MANIFEST}"
+    cp test/e2e/image-snapshot-manifest.yaml ${SHARED_DIR}/${TEST_CSI_DRIVER_MANIFEST}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)